CONTRIBUTING.md

```diff
@@ -182,5 +182,12 @@ When squashing-and-merging:
 -   Do a final review of the one-line PR summary, ensuring that it meets the
     guidelines (e.g., "blob: add more blobbing") and accurately describes the
     change.
+-   If the PR includes a breaking change, which should be declared via a commit
+    with `BREAKING_CHANGE_OK` in it (Travis checks for this), add
```

Comment:
This sentence is a mouthful... can you refactor it to mention what BREAKING_CHANGE_OK is in its own bullet?

Reply:
Done.